### ISSUE TITLE
Send notification to patient through ABDM when thier patient records are edited

### ISSUE THREAD

**Issue by khavinshankar** [ohcnetwork/care_abdm#1]

**Comment by siva222003** [comment]
Hey @khavinshankar, if this issue is still open, can I work on it?

**Comment by khavinshankar** [comment]
@siva222003 This issue is still open, can you comment your solution and eta.

**Comment by siva222003** [comment]
@khavinshankar Can we use signals for this?

**Comment by khavinshankar** [comment]
@siva222003 No, the idea was to check if there is any api in ABDM that supports this, so approach this from that angle

**Comment by siva222003** [comment]
@khavinshankar Thank you for the clarification. I’ve DM’ed you about this yesterday on Slack. Could you check it, or should I repeat it here?

**Comment by vishaljoshi789** [comment]
hey @siva222003, I have gone through the [Building HIP System _ API Flow_M2](https://sandboxcms.abdm.gov.in/uploads/Building_HIP_System_API_Flow_M2_Documentation_d3949bd2_92987fa330.pdf) here in page 19 they have used **Send SMS to Patients (from HIP)**, not sure if this is the complete solution. Hope it helps you.

cc: @khavinshankar

**Comment by siva222003** [comment]
Thank you, @vishaljoshi789, for your help. I have already checked that but am waiting for confirmation from @khavinshankar before starting work.

**Comment by Rustix69** [comment]
@khavinshankar Can I pick up this Issue ? I already worked with this part in send OTP part .
